doc/index.rst

```diff
@@ -21,16 +20,11 @@ Panel makes it simple to make:
 -  Data-rich Python-backed web servers
 -  and anything in between
 
-Panel objects are reactive, immediately updating to reflect changes to
-their state, which makes it simple to compose viewable objects and link
-them into simple, one-off apps to do a specific exploratory task. The
-same objects can then be reused in more complex combinations to build
-more ambitious apps, while always sharing the same code that works well
-on its own.
+Panel objects are reactive, immediately updating to reflect changes to their state, which makes it simple to compose viewable objects and link them into simple, one-off apps to do a specific exploratory task. The same objects can then be reused in more complex combinations to build more ambitious apps, while always sharing the same code that works well on its own.
 
-Compared to other tookits for building interactive web apps, Panel is unique in that lets you move the same code freely between an interactive `Jupyter Notebook <http://jupyter.org>`__ prompt and a fully deployable standalone server.  That way you can easily switch between exploring your data, building visualizations, adding custom interactivity, sharing with non-technical users, and back again at any point, using the same tools and the same code throughout. Panel thus helps support your entire workflow, so that you never have to commit to only one way of using your data and your analyses, and don't have to rewrite your code just to make it usable in a different way.
+Panel lets you move the same code freely between an interactive `Jupyter Notebook <http://jupyter.org>`__ prompt and a fully deployable standalone server.  That way you can easily switch between exploring your data, building visualizations, adding custom interactivity, sharing with non-technical users, and back again at any point, using the same tools and the same code throughout. Panel thus helps support your entire workflow, so that you never have to commit to only one way of using your data and your analyses, and don't have to rewrite your code just to make it usable in a different way.
 
-Some example Panel apps: (click on the title to see the code in each case)
+Some example Panel apps: (Click on the title to see the code, or the image to deploy on mybinder if it has resources available.)
```

Comment:
The errors are not actually related to resources I think it's a race condition between the binder starting bokeh server and the client connecting to it, but I'm okay not explaining that and eventually replacing binder with AE or AWS deployments.

Reply:
Sure. I was trying to think of a sufficiently vague way to say that errors should be expected, without getting into details. :-) I'll merge as-is, but if you can think of a short way to explain the issue better, please do! (And if you can think of a way to avoid the race condition, that would be even better!)